src/index.css

```diff
@@ -1,4 +1,294 @@
+  font-size: 1.375rem;
+  font-weight: bold;
+  font-stretch: normal;
+  font-style: normal;
+  line-height: normal;
+  letter-spacing: -0.2px;
+  color: #121212;
+  margin-top: 0;
+  margin-bottom: 0;
+}
+
+.bookContainer > p {
+  font-family: 'Roboto Slab', serif;
+  font-size: 0.875rem;
+  font-weight: 300;
+  font-stretch: normal;
+  font-style: normal;
+  line-height: normal;
+  letter-spacing: normal;
+  color: #4386bf;
+  margin-bottom: 1.313rem;
+  margin-top: 0;
+}
+
+.actionsContainer > button {
+  font-family: 'Roboto Slab', serif;
+  font-size: 0.875rem;
+  font-weight: 300;
+  font-stretch: normal;
+  font-style: normal;
+  line-height: normal;
+  letter-spacing: normal;
+  color: #4386bf;
+  background-color: #fff;
+  padding-top: 0.188rem;
+  padding-bottom: 0.125rem;
+}
+
+.commentsBtn {
+  padding-right: 0.938rem;
+  border: none;
+  border-right: 1px solid #e8e8e8;
+}
+
+.removeBtn {
+  padding-left: 1.063rem;
+  padding-right: 0.938rem;
+  border: none;
+  border-right: 1px solid #e8e8e8;
+}
+
+.editBtn {
+  padding-left: 1.063rem;
+  border: none;
+}
+
+.progressContainer {
+  display: flex;
+  justify-content: space-between;
+  width: 23.625rem;
+  align-items: center;
+}
+
+.circleProgress {
+  display: flex;
+  align-items: center;
+  padding-right: 30px;
+  border-right: 1px solid #e8e8e8;
+}
+
+.circle {
+  height: 4.25rem;
+}
+
+.circleProgressText {
+  display: flex;
+  flex-direction: column;
+}
+
+.circleProgressText > p {
+  margin-top: 0;
+  margin-bottom: 0;
+  margin-left: 0.68rem;
+}
+
+.progressT1 {
+  font-family: 'Montserrat', sans-serif;
+  font-size: 2rem;
+  font-weight: normal;
+  font-stretch: normal;
+  font-style: normal;
+  line-height: normal;
+  letter-spacing: normal;
+  color: #121212;
+}
+
+.progressT2 {
+  opacity: 0.5;
+  font-family: 'Montserrat', sans-serif;
+  font-size: 0.875rem;
+  font-weight: normal;
+  font-stretch: normal;
+  font-style: normal;
+  line-height: normal;
+  letter-spacing: normal;
+  color: #121212;
+}
+
+.chapterProgress {
+  display: flex;
+  flex-direction: column;
+}
+
+.chapterText {
+  opacity: 0.5;
+  font-family: 'Roboto Slab', serif;
+  font-size: 0.813rem;
+  font-weight: 300;
+  font-stretch: normal;
+  font-style: normal;
+  line-height: normal;
+  letter-spacing: normal;
+  color: #121212;
+  margin-top: 0;
+  margin-bottom: 0;
+}
+
+.chapterNo {
+  font-family: 'Roboto Slab', serif;
+  font-size: 1rem;
+  font-weight: 300;
+  font-stretch: normal;
+  font-style: normal;
+  line-height: normal;
+  letter-spacing: -0.4px;
+  color: #121212;
+  margin-top: 0.438rem;
+  margin-bottom: 1.438rem;
+}
+
+.updateBtn {
+  font-family: 'Roboto Slab', serif;
+  font-size: 0.813rem;
+  font-weight: 300;
+  font-stretch: normal;
+  font-style: normal;
+  line-height: normal;
+  letter-spacing: 0.5px;
+  text-align: center;
+  color: #fff;
+  padding: 0.438rem 1.188rem 0.5rem 1.375rem;
+  border-radius: 3px;
+  background-color: #0290ff;
+  border-width: 0;
+}
```

Comment:
- [ ] Kindly transform this text too because of the above mentioned reason 👆 of getting a replica of the Zeplin's design as mentioned in the project requirements ✌✔